rpc/errors.go

```diff
@@ -18,6 +18,15 @@ package rpc
 
 import "fmt"
 
+var (
+	_ Error = new(methodNotFoundError)
```

Comment:
what's this??

Reply:
Not sure why they added this here, but I'm guessing it's their way of ensuring at compile-time that these types satisfy the `rpc.Error` interface.  Here are some other places where similar things are done:

https://github.com/ethereum/go-ethereum/blob/faacc8e0faa89f5b3a7055e86924a95b6a9be944/contracts/checkpointoracle/contract/oracle.go#L18-L27
https://github.com/ethereum/go-ethereum/blob/faacc8e0faa89f5b3a7055e86924a95b6a9be944/ethclient/ethclient_test.go#L43-L56
https://github.com/ethereum/go-ethereum/blob/faacc8e0faa89f5b3a7055e86924a95b6a9be944/metrics/metrics_test.go#L14-L18